packages/gafl-webapp-service/src/pages/multibuy/remove-licence/__tests__/update-transaction.spec.js

```diff
@@ -76,79 +74,96 @@ describe('remove-licence > update transaction', () => {
     jest.clearAllMocks()
   })
 
-  it('status.setCurrentPermission is being called with different permission', async () => {
-    const setPermission = getStatusPermission()
-    const mockRequest = createMockRequest({
-      cache: { status: { permissions: [getStatusPermission(), setPermission, getStatusPermission({ [REMOVE_LICENCE.page]: true })] } }
-    })
-    await updateTransaction(mockRequest)
-    expect(mockRequest.cache().helpers.status.setCurrentPermission()).toEqual(expect.objectContaining(setPermission))
-  })
-
-  it('transaction.set is being called with a permission removed', async () => {
+  it('transaction cache is updated after permission is deleted', async () => {
     const transaction = { permissions: [getTransactionPermissionOne(), getTransactionPermissionTwo(), getTransactionPermissionThree()] }
-    const mockRequest = createMockRequest({ cache: { transaction: transaction } })
+    const mockRequest = getMockRequest({ cache: { transaction: transaction } })
     await updateTransaction(mockRequest)
     expect(mockRequest.cache().helpers.transaction.set()).toEqual(expect.objectContaining(transaction))
   })
 
-  it('page.set is being called with a permission removed', async () => {
+  it('page cache is updated after permission is deleted', async () => {
     const page = { permissions: [getPagePermission({ [REMOVE_LICENCE.page]: true }), getPagePermission(), getPagePermission()] }
-    const mockRequest = createMockRequest({ cache: { page: page } })
+    const mockRequest = getMockRequest({ cache: { page: page } })
     await updateTransaction(mockRequest)
     expect(mockRequest.cache().helpers.page.set()).toEqual(expect.objectContaining(page))
   })
 
-  it('status.set is being called with a permission removed', async () => {
+  it('status cache is updated after permission is deleted', async () => {
+    const statusSet = jest.fn()
     const status = { permissions: [getStatusPermission({ [REMOVE_LICENCE.page]: true }), getStatusPermission(), getStatusPermission()] }
-    const mockRequest = createMockRequest({ cache: { status: status } })
+    const mockRequest = getMockRequest({ cache: { status } }, statusSet)
     await updateTransaction(mockRequest)
-    expect(mockRequest.cache().helpers.status.set()).toEqual(expect.objectContaining(status))
+    expect(statusSet).toHaveBeenCalledWith(status)
   })
 
-  it('addressLookup.set is being called with a permission removed', async () => {
+  it('addressLookup cache is updated after permission is deleted', async () => {
     const addressLookup = { permissions: [{}, {}, {}] }
-    const mockRequest = createMockRequest({ cache: { addressLookup: addressLookup } })
+    const mockRequest = getMockRequest({ cache: { addressLookup: addressLookup } })
```

Comment:
1) addressLookup is `{ permissions: [{}, {}, {}] }` by default
2) `{ addressLookup }`, not `{ addressLookup: addressLookup }`